framework/src/node/network/network.ts

```diff
@@ -409,15 +410,13 @@ export class Network {
 		requestPacket: liskP2P.p2pTypes.P2PRequestPacket,
 	): Promise<liskP2P.p2pTypes.P2PResponsePacket> {
 		return this._p2p.request({
-			procedure: requestPacket.procedure,
-			data: requestPacket.data,
+			...requestPacket,
```

Comment:
do we need to change here?

Reply:
yeah maybe keep it as before to avoid passing any unnecessary props to p2p.request